### PR TITLE
REDCORE-269 Tests webservices API

### PR DESCRIPTION
**Pull request by javigomez**
# Do not merge, this is work in progress

This is the first stage of testing the Webservices in redCORE.

We need to wait for this to be merged: https://github.com/Codegyre/Robo/pull/178 before it works.

And still several parts are missing.

### CODE REVIEW

File: component/admin/views/webservices/tmpl/default.php [hunk 1]
```diff
@@ -322,7 +322,7 @@ class="btn btn-success"
 							</div>
 							<div class="controls">
 								<button
-									class="btn btn-success"
+									class="btn btn-success lc-install_all_webservices"
```
Comment (puneet0191) [inline component/admin/views/webservices/tmpl/default.php]: good approach :+1:

File: .travis.yml [hunk 2]
```diff
@@ -35,9 +35,11 @@ script:
 - php .travis/phpcs.php
 - php .travis/misseddebugcodechecker.php component/ libraries/ modules/ plugins/
 - mv tests/acceptance.suite.dist.yml tests/acceptance.suite.yml
-- php vendor/bin/robo prepare:site-for-system-tests
-- vendor/bin/robo run:groups-tests --group installation
-- vendor/bin/robo run:groups-tests --group installation --skip_group yes
+- vendor/bin/robo prepare:site-for-system-tests
+- vendor/bin/robo run:tests --groups installation
+- vendor/bin/robo run:tests --skip_groups=installation,uninstallation
```
Comment (puneet0191) [inline .travis.yml]: Uninstallation group is also a good way to handle this :+1:  nice work here

File: component/admin/views/webservices/tmpl/default.php (hunk 1 above)
Comment (javigomez) [inline component/admin/views/webservices/tmpl/default.php]: is based on this best practice http://phptest.club/t/codeception-tips-tricks/14/8

File: codeception.yml [hunk 3]
```diff
@@ -9,4 +9,5 @@ settings:
     colors: true
     memory_limit: 1024M
 groups:
-    installation: [tests/acceptance/01-InstallJoomlaCept.php, tests/acceptance/02-InstallExtensionCept.php]
\ No newline at end of file
+    installation: [tests/acceptance/01-InstallJoomlaCept.php, tests/acceptance/02-InstallExtensionCept.php]
+    uninstallation: [tests/acceptance/99-UninstallExtensionCept.php]
```
Comment (puneet0191) [inline codeception.yml]: 99? what was wrong with ZZ

**Comment by javigomez** [comment]
Now that  Codegyre/Robo#178 was fixed

I have added the tests for Contact Webservices:

screen shot 2015-06-18 at 19 27 12

Is still work in progress. Waiting for Travis

**Comment by javigomez** [comment]
The DELETE webservice is not yet working, we need to see what to do with https://redweb.atlassian.net/browse/REDCORE-418

**Pull request closed without merging**